homeassistant/components/opnsense/types.py

```diff
@@ -0,0 +1,15 @@
+"""Types for OPNsense integration."""
+
+from typing import TypedDict
+
+
+class APIData(TypedDict):
+    """API data for OPNsense."""
+
+    api_key: str
+    api_secret: str
+    base_url: str
+    verify_cert: bool
+
+
+type Interfaces = list[str]
```

Comment:
I don't think you have to make a special type for this

Reply:
OK, was just trying to make it a little nicer to read